molo/yourtips/migrations/0003_auto_20170705_1011.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
@cblignaut lets look at the migrations again, I think we need to do some cleanup